doc/governance.rst

```diff
@@ -0,0 +1,95 @@
+Ceph Governance
+===============
+
+The Ceph open source community is guided by a few different groups.
+
+Project Leader
+--------------
+
+The Ceph project currently is led by Sage Weil <sage@redhat.com>.  The
```

Comment:
my own preference: instead of `The Ceph project currently is led by`, I'd have

  `The Ceph project is currently led by`
or
  `The Ceph project, currently, is led by`